pom.xml

```diff
@@ -92,6 +92,10 @@
             <id>rlf-mvn-repo</id>
             <url>https://raw.github.com/rlf/mvn-repo/master/</url>
         </repository>
+		<repository>
```

Comment:
I'd rather not use local file repositories in Maven. Another issue at the moment is that anyone who hasn't bought the plugin is unable to build uSkyBlock with this dependency. It would be nice if the WildStacker developer releases a public API package to use as dependency.

Reply:
The WildStack Api is free. https://www.spigotmc.org/threads/wildstackerapi-api-for-wildstacker-2.338775/
I can remove the local repository, because I changed the dependency

Reply:
Removed the local repository